lib/diagnostic.js

```diff
@@ -14,12 +14,22 @@
 'use strict';
 
 const fs = require('fs'); // jshint ignore: line
+const inFile = process.argv[2];
 
-const sumLines = (filename, callback) => {
-  /* your response here */
-  callback();
+const sumLines = ('integers.txt', callback) => {
```

Comment:
This function needs to take in 2 arguments. Predetermining what file your function will read will disable it from reading anything else.